packages/components/src/components/link/link.stories.ts

```diff
@@ -2,19 +2,24 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes } = storybookDefaults('sd-link');
 const { generateTemplate } = storybookTemplate('sd-link');
 const { overrideArgs } = storybookHelpers('sd-link');
 
+/**
+ *
+ * This component is used to display links and can also be used to perform actions like downloading files.
```

Comment:
Allows users to navigate to another location or perform actions like e.g. downloading files.

Reply:
Updated